packages/babel-plugin-transform-es2015-block-scoping/test/fixtures/general/switch/expected.js

```diff
@@ -9,3 +11,17 @@ switch (a) {
     var d = 3;
     break;
 }
+
+switch (true) {
+  case true:
+    var _a = false;
+    var _b = true;
+    var _c = 4;
+    var _d = 5;
+
+  case false:
+    class _e {}
```

Comment:
e.g. they to show they aren't the same, even though the transpiler might today handle them the same if they have the same idenifiers

![image](https://cloud.githubusercontent.com/assets/762949/14934503/c6c6134e-0e66-11e6-8b46-9a53e2bab400.png)